notebooks/125-torchvision-zoo-to-openvino/125-convnext-classification.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>Could you please add a few more lines here. Just to elaborate a little on torchvision models hub. What models are available and so on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
It is added in README (`The `torchvision.models` subpackage contains definitions of models for addressing different tasks, including: image classification, pixelwise semantic segmentation, object detection, instance segmentation, person keypoint detection, video classification, and optical flow.`). Here is only example for a classification model. DO you think I should duplicate this information here or something else?

Reply:
I think it makes sense. It is just a couple of lines and those people who did not read readme will have an additional anchor

Reply:
Added